src/models/core/SearchConfig.ts

```diff
@@ -40,10 +40,10 @@ export interface BaseAnswersConfig {
 
 /**
  * Configuration options for {@link AnswersCore}, which includes the
```

Comment:
are these links to `AnswersCore` updated in one of the other PRs?

Reply:
They will be added in another PR.  Since I renamed SearchCore in an old PR and not in this one, it would point to nothing and cause an error :(